File: acmp.ru/Python/[0051] - [0100]/[0068] Дом - Школа - Дом.py
sp = input()
x = int(input())

if sp == 'Home':
    print('Yes')
else:
    if x % 2 == 0:
        print('No')
    else:
        print('Yes')

# print('Yes' if input() == 'Home' or int(input()) % 2 else 'No')

File: acmp.ru/[0001] - [0100]/[0068] Дом - Школа - Дом.cpp
#include <iostream>
#include <cstring>
using namespace std;


int main() {
    string s;
    int x;
    
    cin >> s >> x;

    if (s == "Home" or x % 2 != 0) {
        cout << "Yes" << endl;
    } else {
        cout << "No" << endl;
    }

    return 0;
}